test/propagated_params_test.go

```diff
@@ -165,7 +165,6 @@ metadata:
   name: propagated-parameters-fully
   namespace: %s
 spec:
-  serviceAccountName: default
```

Comment:
It might still be in the "expected" pipelinerun though (which is also something we need to "ignore")

Reply:
makes sense, i've ignored the field in TaskRunSpec and PipelineRunSpec as well